mongodbatlas/version.go

```diff
@@ -0,0 +1,15 @@
+package mongodbatlas
+
+import "fmt"
+
+const (
+	// UserAgent the default user agent string reported by this client
+	UserAgent = "go-mongodbatlas"
```

Comment:
I think we should not export this
```suggestion
	userAgent = "go-mongodbatlas"
```